pkg/sql/conn_executor_exec.go

```diff
@@ -835,6 +836,16 @@ func (ex *connExecutor) execStmtInOpenState(
 			return makeErrEvent(err)
 		}
 		return nil, nil, nil
+
+	case *tree.CreateFunction:
```

Comment:
I don't think we need to make this into the conn executor. Do you want to give it a shot in `optbuilder/create_function.go`?